File: old/apps/All_Hallows_Read/All_Hallows_Read/WebViewController.m
//
//  WebViewController.m
//  All_Hallows_Read
//
//  Created by Samuel Svenningsen on 10/17/14.
//  Copyright (c) 2014 Sam Svenningsen. All rights reserved.
//

#import "WebViewController.h"

@interface WebViewController ()
@property (weak, nonatomic) IBOutlet UIWebView *faqWebView;

@end

@implementation WebViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    self.faqWebView.delegate = self;
    
    NSURL *url = [NSURL URLWithString:@"http://www.allhallowsread.com/the-faq/"];
    NSURLRequest *requestURL = [NSURLRequest requestWithURL:url];
    [self.faqWebView loadRequest:requestURL];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

@end

File: old/apps/Training/RedditClient - Lecture 8/RedditClient/WebViewController.swift
//
//  WebViewController.swift
//  RedditClient
//
//  Created by Eric Appel on 11/10/14.
//  Copyright (c) 2014 Eric Appel. All rights reserved.
//

import UIKit

class WebViewController: UIViewController {
    
    var url: NSURL!

    @IBOutlet weak var webView: UIWebView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.title = url.absoluteString
        
        let webView = UIWebView(frame: view.frame)
        webView.scalesPageToFit = true
        view.addSubview(webView)
        
        let request = NSURLRequest(URL: url)
        webView.loadRequest(request)
    }
}
